src/Api/Controllers/OrganizationUsersController.cs

```diff
@@ -452,6 +456,29 @@ public async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkRest
         return await RestoreOrRevokeUsersAsync(orgId, model, (orgId, orgUserIds, restoringUserId) => _organizationService.RestoreUsersAsync(orgId, orgUserIds, restoringUserId, _userService));
     }
 
+    [HttpPatch("enable-secrets-manager")]
+    [HttpPut("enable-secrets-manager")]
+    public async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkEnableSecretsManagerAsync(Guid orgId,
+        [FromBody] OrganizationUserBulkRequestModel model)
+    {
+        if (!await _currentContext.ManageUsers(orgId))
+        {
+            throw new NotFoundException();
+        }
+
+        var orgUsers = (await _organizationUserRepository.GetManyAsync(model.Ids))
+            .Where(ou => ou.OrganizationId == orgId).ToList();
+        if (orgUsers.Count == 0)
+        {
+            throw new BadRequestException("Users invalid.");
```

Comment:
I would have expected this to be a 404 not found. Is there anything specific we are trying to accomplish by using `BadRequestException`?

Reply:
I was going with what was pre-established by the other bulk requests. 

Specifically, the bulk delete.

https://github.com/bitwarden/server/blob/ca7ced4e435823edc677efcced06f5689a2e8754/src/Core/Services/Implementations/OrganizationService.cs#L1640-L1647

Reply:
Makes sense, thanks for pointing that out. As usual we can leave it for consistency and address the responses in a different PR if we want.